### DISCUSSION TITLE
Accessing WebUI

### DISCUSSION THREAD

**Discussion by Alaa-Aim** [vortigont/espem#32] [Q&A] [answered]
Hi @vortigont , in the early version, accessing the WebUI can be simply by typing 'http://espem/?b_espem' , or 'http://espem/getdata' to get raw data, in the current version and previous one, you have to know the IP of the ESP32, so you can access it, i.e. 'http://192.168.1.110//?b_espem' , can this be fixed ? for easy access and convenient, thanks in advanced

**Comment by vortigont** [comment, marked as answer]
@Alaa-Aim 
actually the software registers it's hostname when acquiring DHCP address. Pls, check WiFi setting to find it's current hostname, it might be some autogenerated embui-xxxx. You can change it to the one you like.

**Reply by Alaa-Aim** [reply]
@vortigont , many thanks
